strongbox-commons/src/main/java/org/carlspring/strongbox/io/ReplacingInputStream.java

```diff
@@ -0,0 +1,102 @@
+package org.carlspring.strongbox.io;
+
+import java.io.FilterInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Deque;
+import java.util.Iterator;
+import java.util.LinkedList;
+
+
+public class ReplacingInputStream extends FilterInputStream {
```

Comment:
* **Cosmetic:** This is not formatted according to the convention
* What is this class used for? Would you mind explaining what you're doing?